src/java.base/share/classes/java/lang/StableValue.java

```diff
@@ -727,8 +726,8 @@ static <E> List<E> list(int size,
      * is rethrown to the initial caller and no value associated with the provided key
      * is recorded.
      * <p>
-     * Any direct {@link Map#values()} or {@link Map#entrySet()} views
-     * of the returned map are also stable.
+     * Any {@link Map#values()} or {@link Map#entrySet()} views of the returned map are
+     * also stable.
```

Comment:
You resolved https://github.com/openjdk/jdk/pull/23972#discussion_r2058903119, so the following lines can also get rid of "direct":
https://github.com/openjdk/jdk/blob/fbc4691bfa11f31601fd89d05da63e689343e214/src/java.base/share/classes/java/lang/StableValue.java#L687-L688